alexandrie/src/api/crates/publish.rs

```diff
@@ -289,16 +289,6 @@ pub(crate) async fn put(mut req: Request<State>) -> tide::Result {
                 }));
             }
 
-            //? Is the version higher than the latest known one?
-            let latest = state.index.latest_record(krate.name.as_str())?;
-            if crate_desc.vers <= latest.vers {
```

Comment:
Hhhm I think this check (the <=) might have been the only one preventing you to accidentally clobber an existing version?

I guess there's no problem if we're careful, but I'm curious what happens if someone accidentally tried to upload a -beta1 that already exists after this patch :thinking:

Reply:
that's a fair point, let's keep this one check and change the error message